examples/kotlin/src/main/kotlin/com/example/authors/QueriesImpl.kt

```diff
@@ -0,0 +1,109 @@
+// Code generated by sqlc. DO NOT EDIT.
+
+package com.example.authors
+
+import java.sql.Connection
+import java.sql.SQLException
+
+const val createAuthor = """-- name: createAuthor :one
+INSERT INTO authors (
+          name, bio
+) VALUES (
+  ?, ?
+)
+RETURNING id, name, bio
+"""
+
+data class CreateAuthorParams (
+  val name: String,
+  val bio: String?
+)
+
+const val deleteAuthor = """-- name: deleteAuthor :exec
+DELETE FROM authors
+WHERE id = ?
+"""
```

Comment:
use `trimIndent` or `trimMargin` so you can indent?

Reply:
I could, but why bother?